torch/jit/__init__.py

```diff
@@ -554,7 +554,7 @@ def graph_diagnostic_info():
                     check_tensor_val = n_check.t('value')
 
                     try:
-                        torch.testing.assert_allclose(mod_tensor_val, check_tensor_val)
+                        torch.testing.assert_allclose(mod_tensor_val.to(torch.double), check_tensor_val.to(torch.double))
```

Comment:
this can't be necessary, right?  Because you are now doing the conversion in the function call as well (which is wrong, see below).

Reply:
ah, I may have changed this location first and then changed the below when this alone didn't fix my issue.

Reply:
changed isclose and got rid of these conversions